sdk/appcenter-distribute-play/src/main/res/values/appcenter_distribute.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?><!--
+  ~ Copyright (c) Microsoft Corporation. All rights reserved.
+  ~ Licensed under the MIT License.
+  -->
+
+<resources xmlns:tools="http://schemas.android.com/tools">
+    <string name="appcenter_distribute_update_dialog_title">App update available</string>
+    <string name="appcenter_distribute_update_dialog_message_optional">%1$s %2$s (%3$d) is available to download and install.</string>
+    <string name="appcenter_distribute_update_dialog_message_mandatory">The developers of this app require you to update to %1$s %2$s (%3$d).</string>
```

Comment:
Should we create a distribute-core to share identical files and avoid duplication?

Reply:
I don't think that we should add one more library and publish one more package to share just a few files. Sharing files like Xamarin would work here, but there is no similar way to do it in native Android

Reply:
We need to try symlinks for these files

Reply:
Are symbolic links working on Windows? If not we should not use them and keep this repo compatible with all OS.

Reply:
I checked changes on Windows. Looks like the symbolic links for Mac don't work on Windows, I reverted changes.